functional_tests/functional/testdata/python/deployment.yaml

```diff
@@ -19,6 +19,6 @@ spec:
       containers:
         - image: quay.io/splunko11ytest/python_test:latest
           name: python-test
-          imagePullPolicy: IfNotPresent
+          imagePullPolicy: Always
```

Comment:
Did you run into issues with `IfNotPresent` here?

Reply:
ah this was meant for local test, going to revert